rpi-node/README.md

```diff
@@ -0,0 +1,75 @@
+# Supported tags and respective `Dockerfile` links
+
+-	[`0.10.36`, `0.10` (*0.10/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.10/Dockerfile)
+-	[`0.10.36-onbuild`, `0.10-onbuild` (*0.10/onbuild/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0a0c83fa341710b6a55a07c3eadc9a44a6bc84fe/0.10/onbuild/Dockerfile)
+-	[`0.10.36-slim`, `0.10-slim` (*0.10/slim/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.10/slim/Dockerfile)
+-	[`0.10.36-wheezy`, `0.10-wheezy` (*0.10/wheezy/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.10/wheezy/Dockerfile)
+-	[`0.12.0`, `0.12`, `0`, `latest` (*0.12/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/Dockerfile)
+-	[`0.12.0-onbuild`, `0.12-onbuild`, `0-onbuild`, `onbuild` (*0.12/onbuild/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0a0c83fa341710b6a55a07c3eadc9a44a6bc84fe/0.12/onbuild/Dockerfile)
+-	[`0.12.0-slim`, `0.12-slim`, `0-slim`, `slim` (*0.12/slim/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/slim/Dockerfile)
+-	[`0.12.0-wheezy`, `0.12-wheezy`, `0-wheezy`, `wheezy` (*0.12/wheezy/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/wheezy/Dockerfile)
+-	[`0.8.28`, `0.8` (*0.8/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0de2ce005deb23a5ec9eaa667c5f6128ec323f79/0.8/Dockerfile)
```

Comment:
There is a minor fix in Dockerfiles except for onbuild. That's why the commits are different.